signac_dashboard/dashboard.py

```diff
@@ -388,6 +388,10 @@ def _render_job_view(self, *args, **kwargs):
         else:
             return self._render_error(ValueError(f"Invalid view mode: {view_mode}"))
 
+    def _render_project_view(self, *args, **kwargs):
+        g.active_page = "project"
+        return render_template("project_info.html", *args, **kwargs)
```

Comment:
I would look in `tests/test_dashboard.py` and add very basic tests that the project page can render, and to ensure that the modules can be loaded. Look at `test_get_jobs` for an example. That would help fix some of the coverage issues. (Real web application testing would use Selenium or something like that but is far too complicated for this project. We just want to make sure the Python code can be executed and the server can render the page.)